docs/internals/git-workflow.md

```diff
@@ -111,10 +111,10 @@ review your suggestion, and provide appropriate feedback along the way.
 
 > For small changes or documentation issues or straightforward fixes, you don't need to create an issue, a pull request is enough in this case.
 
-### 2. Fetch the latest code from the main Yii branch
+### 2. Pull the latest code from the main Yii branch
 
 ```
-git fetch upstream
+git pull upstream
```

Comment:
also pull requires specifying a branch and might merge the upstream branch with your local branch which is not what you'd expect here unless you are on master right now.